### ISSUE TITLE
Don't stop the test on fail

### ISSUE THREAD

**Issue by cacharle** [nesvoboda/socrates#7]
When a fail occurs the test should continue the next tests to show more feed back to the user.

For the infinity test, the user would be able to see if the fail is an isolated incident or if it fails every time

**Comment by arseniycodes** [comment]
Yeah, I agree. I only made it stop on error because I wasn't sure I would be able to indicate the error clearly enough.
